src/Nancy.Serialization.JsonNet/JsonNetBodyDeserializer.cs

```diff
@@ -51,6 +51,8 @@ public bool CanDeserialize(MediaRange mediaRange, BindingContext context)
         /// <returns>Model instance</returns>
         public object Deserialize(MediaRange mediaRange, Stream bodyStream, BindingContext context)
         {
+            bodyStream.Position = 0;

```

Comment:
@vbfox be sure to update everything as we've just pulled in stuff in all the serializer repos :D